src/components/Dashboard/Tiles/SmallTile.jsx

```diff
@@ -1,21 +1,65 @@
-import React from 'react';
+import { TrashIcon } from '@heroicons/react/outline';
+import React, { useState } from 'react';
 
-const SmallTile = ({ title, img, url }) => {
+import { useAuth } from '../../../contexts/AuthContextProvider';
+import { deleteDoc, doc } from 'firebase/firestore';
+import { db } from '../../../utils/FirebaseConfig';
+
+const SmallTile = ({ title, img, url, deleteBookmark }) => {
+	const [fallback, setFallback] = useState(false);
+	const { user } = useAuth();
 	const handleChangeUrl = () => {
 		window.api.send('searchURL', url);
 	};
 
+	const handleDeleteBookmark = () => {
+		deleteDoc(doc(db, `users/${user.uid}/bookmarks/${title}`));
+	};
```

Comment:
Dit is wel heel specifieke code voor SmallTile component. Mss component hernoemen naar BookmarkTile ?